ogre/src/OgreMaterial.cc

```diff
@@ -660,9 +660,9 @@ void OgreMaterial::SetDepthMaterial(const double _far,
 
   // TODO(anyone): convert depth configuration into a ShaderType
   // Get shader parameters path
-  const char *env = std::getenv("IGN_RENDERING_RESOURCE_PATH");
+  const char *env = std::getenv("GZ_RENDERING_RESOURCE_PATH");
   std::string resourcePath = (env) ? std::string(env) :
-      IGN_RENDERING_RESOURCE_PATH;
+      GZ_RENDERING_RESOURCE_PATH;
```

Comment:
Ah just found another env var in need of migration

Reply:
https://github.com/gazebosim/gz-rendering/pull/629/commits/c69cbc52219181c67458a53e95f45161a4e6398b